### PR TITLE
ixwebsocket: add version 11.0.9

### PR DESCRIPTION
**Pull request by liboyue**
Specify library name and version:  **ixwebsocket/11.0.8** and **ixwebsocket/11.0.9**

@prince-chrismc  I updated the OpenSSL, now you can close https://github.com/conan-io/conan-center-index/pull/4828 and merge this pr. The 11.0.8 is a "major" release according to the author, so I added it too. Thanks!

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/ixwebsocket/all/conandata.yml [hunk 1]
```diff
@@ -17,3 +17,9 @@ sources:
   "11.0.4":
     url: "https://github.com/machinezone/IXWebSocket/archive/v11.0.4.tar.gz"
     sha256: "deebfa04bd6bd930b2f7c0daba8674c8e8bc273a3d735b1877b1f4bb2c0e8596"
+  "11.0.8":
+    url: "https://github.com/machinezone/IXWebSocket/archive/v11.0.8.tar.gz"
+    sha256: "5d02fa5a23427e07326b4bbd285479aabb18fa2e1d47c05a676523ae4d6a4dca"
```
Comment (prince-chrismc) [inline recipes/ixwebsocket/all/conandata.yml:22]:
```suggestion
```

File: recipes/ixwebsocket/config.yml [hunk 2]
```diff
@@ -11,3 +11,7 @@ versions:
     folder: all
   "11.0.4":
     folder: all
+  "11.0.8":
+    folder: all
```
Comment (prince-chrismc) [inline recipes/ixwebsocket/config.yml:15]:
```suggestion
```

**Review by prince-chrismc — CHANGES_REQUESTED** [review]
We try to avoid adding older versions, since they are almost never used.

If you specifically need that version that's acceptable as well!

**Comment by liboyue** [comment]
Thanks for the comment. However, as in the release page https://github.com/machinezone/IXWebSocket/releases, 11.0.8 is still labeled as "latest", and I was using 11.0.8 too. So I guess it may be better to add this version.

**Comment by prince-chrismc** [comment]
If you look at the release history the mainters have been very inconsistent at creating proper GitHub releases but the tags and teh official changlog have all the versions https://machinezone.github.io/IXWebSocket/CHANGELOG/#1109-2021-03-07

I also noticed this going through the repro
https://github.com/machinezone/IXWebSocket/commit/ee5a2eb46ee0e109415dc02b0db85a9c76256090
Please please adjust this in the recipe

**Comment by liboyue** [comment]
Should I remove this check https://github.com/conan-io/conan-center-index/blob/master/recipes/ixwebsocket/all/conanfile.py#L51 since it does not have a minimum requirement or change it to `11`?

**Comment by prince-chrismc** [comment]
Maybe?

`tools.check_min_cppstd(self, 14 if tools.Version(self.version) < "11.0.8" else 11)`

**Review by prince-chrismc — APPROVED** [review]

**Comment by prince-chrismc** [comment]
Once a week new comments on the EAP issue are added automatically so it should not be too long before this can continue =)

**Pull request closed without merging**